README.md

```diff
@@ -57,3 +55,18 @@ bin/dev jstest [files]
 ```shell
 bin/dev check
 ```
+## Updating
+
+1. Occasionally, newer versions of the devobx container will be made. Your invocation of `bin/dev backend` will likely pull it, but you can manually do so by running.
+```shell
+bin/dev pull
+```
+
+## Fix Python Location for vscode
+
+```shell
+bin/dev run bash
```

Comment:
I gave up trying to make this work in bash script. You'll just have to copy paste couple lines if you want it.